src/rock/models/likes/resolver.js

```diff
@@ -0,0 +1,12 @@
+
+// import { flatten } from "lodash";
+
+export default {
+  Mutation: {
+    toggleLike(_, { nodeId }, { models, user }) {
+      // XXX what should the response be if invalid/insufficient data?
```

Comment:
@JakeDawkins we have a generic mutation response that we should implement here. https://github.com/NewSpring/Heighliner/blob/30577753b3805a1dc31dd07ef28383cfdfe94d9d/src/util/node/schema.js#L7-L11